packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -0,0 +1,388 @@
+	const response = await rpcMethods.getTransactionCount(
+		web3Context.requestManager,
+		address,
+		blockNumber ?? web3Context.defaultBlock,
+	);
+
+	return convertToValidType(
+		response,
+		returnType ?? web3Context.defaultReturnType,
+	) as ValidReturnTypes[ReturnType];
+}
+
+// TODO Needs to convert input to hex string
+// public async sendTransaction(transaction: Transaction) {
+// 	return rpcMethods.sendTransaction(this.web3Context.requestManager, transaction);
+// }
+
+export const sendSignedTransaction = async (
+	web3Context: Web3Context<EthExecutionAPI>,
+	transaction: HexStringBytes,
+) => rpcMethods.sendRawTransaction(web3Context.requestManager, transaction);
+
+// TODO address can be an address or the index of a local wallet in web3.eth.accounts.wallet
+// https://web3js.readthedocs.io/en/v1.5.2/web3-eth.html?highlight=sendTransaction#sign
+export const sign = async (
+	web3Context: Web3Context<EthExecutionAPI>,
+	message: HexStringBytes,
+	address: Address,
+) => rpcMethods.sign(web3Context.requestManager, address, message);
+
+// TODO Needs to convert input to hex string
+// public async signTransaction(transaction: Transaction) {
+// 	return rpcMethods.signTransaction(this.web3Context.requestManager, transaction);
+// }
+
+// TODO Decide what to do with transaction.to
+// https://github.com/ChainSafe/web3.js/pull/4525#issuecomment-982330076
+// public async call(
+// 	transaction: Transaction & { to: Address },
+// 	blockNumber: BlockNumberOrTag = this.web3Context.defaultBlock,
+// ) {
+// 	return rpcMethods.call(this.web3Context.requestManager, transaction, blockNumber);
+// }
+
+// TODO Missing param
+export async function estimateGas<ReturnType extends ValidTypes = ValidTypes.HexString>(
+	web3Context: Web3Context<EthExecutionAPI>,
+	transaction: Partial<TransactionWithSender>,
+	blockNumber: BlockNumberOrTag = web3Context.defaultBlock,
+	returnType?: ReturnType,
+) {
+	const response = await rpcMethods.estimateGas(
+		web3Context.requestManager,
+		transaction,
+		blockNumber,
+	);
+
+	return convertToValidType(
+		response,
+		returnType ?? web3Context.defaultReturnType,
+	) as ValidReturnTypes[ReturnType];
+}
+
+export const getPastLogs = async (web3Context: Web3Context<EthExecutionAPI>, filter: Filter) =>
+	rpcMethods.getLogs(web3Context.requestManager, {
+		...filter,
+		// These defaults are carried over from 1.x
+		// https://web3js.readthedocs.io/en/v1.5.2/web3-eth.html?highlight=sendTransaction#getpastlogs
+		fromBlock: filter.fromBlock ?? web3Context.defaultBlock,
+		toBlock: filter.toBlock ?? web3Context.defaultBlock,
+	});
+
+export const getWork = async (web3Context: Web3Context<EthExecutionAPI>) =>
+	rpcMethods.getWork(web3Context.requestManager);
+
+export const submitWork = async (
+	web3Context: Web3Context<EthExecutionAPI>,
+	nonce: HexString8Bytes,
+	seedHash: HexString32Bytes,
+	difficulty: HexString32Bytes,
+) => rpcMethods.submitWork(web3Context.requestManager, nonce, seedHash, difficulty);
+
+// TODO
```

Comment:
this PR is already growing large, so good to create new issue with all TODOs in that, for tracking and planning in next sprint.

Reply:
Specified [here](https://github.com/ChainSafe/web3.js/pull/4718#issuecomment-1032134238), if this PR is okay to be merged with these outstanding `TODO`s, than I will move them into an issue

Reply:
we should create issue for these [todo](https://github.com/ChainSafe/web3.js/pull/4718#issuecomment-1032134238) s,